UnityProject/Assets/Scripts/Communications/SignalReceiver.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Managers;
+using Mirror;
+using UnityEngine;
+using ScriptableObjects.Communications;
+
+namespace Communications
+{
+	public class SignalReceiver : NetworkBehaviour
+	{
+		public SignalType SignalTypeToReceive = SignalType.PING;
+		[SyncVar] public float Frequency = 122F;
+		[SyncVar] public SignalEmitter Emitter;
+		public float DelayTime = 3f; //How many seconds of delay before the SignalRecieve logic happens for weak signals
+
+
+		private void Awake()
+		{
+			if(isClient) return;
```

Comment:
This doesnt do what you think it does, use `CustomNetworkManager.Instance.IsServer == false`, but that cannot be used in awake, doesnt use awake to add the receiver